.gitattributes

```diff
@@ -1 +1,10 @@
 *.sh eol=lf
+*.md eol=lf
+*.json eol=lf
+*.yaml eol=lf
+.*ignore eol=lf
+*.markdown eol=lf
+nvm-exec eol=lf
+Makefile eol=lf
+bash_completion eol=lf
+
```

Comment:
hmmmm ... if you think we'll have only test files inside the repo, I think - yes, what do you think?